tests/unit/Infrastructure/Manage/Client/DeleteEntityClientTest.php

```diff
@@ -117,4 +118,33 @@ public function provideInvalidProtocols()
         yield ['oauth_ccc'];
         yield ['i-dont-exist'];
     }
+
+    /**
+     * @dataProvider provideExpectedProtocolMappings
+     */
+    public function test_it_maps_enitty_types_correctly_with_existing_protocols($inputProtocol, $expectedProtocol)
+    {
+        $client = m::mock(HttpClientInterface::class);
+        $this->client = new DeleteManageEntityClient(
+            $client,
+            $this->logger
+        );
+
+        $manageEntityId = 'db2e5c63-3c54-4962-bf4a-d6ced1e9cf33';
+        $client
+            ->shouldReceive('delete')
+            ->with(sprintf('/manage/api/internal/metadata/%s/%s', $expectedProtocol, $manageEntityId))
+            ->once()
+            ->andReturn(true);
+
+        $this->client->delete($manageEntityId, $inputProtocol);
+    }
```

Comment:
To make the test even more betterest, consider using the `MockeryPHPUnitIntegration` in this trait. 

https://docs.mockery.io/en/latest/reference/phpunit_integration.html

Reply:
it was already extending the class :p
https://github.com/SURFnet/sp-dashboard/blob/741ee78c8d51024a10ca008411f896b408a1ccb5/tests/unit/Infrastructure/Manage/Client/DeleteEntityClientTest.php#L39

Reply:
Completely missed that!